projects/media-viewer/src/lib/store/selectors/bookmarks.selectors.ts

```diff
@@ -30,12 +36,33 @@ export const getEditableBookmark = createSelector(
 export const getBookmarkInfo = createSelector(
   fromAnnotations.getDocumentIdSetId,
   fromDocument.getPdfPosition,
-  (docSetId, pdfPosition) => {
+  fromDocument.getPages,
+  (docSetId, pdfPosition, pages) => {
     return {
       pageNumber: pdfPosition.pageNumber - 1,
       xCoordinate: pdfPosition.left,
-      yCoordinate: pdfPosition.top,
+      yCoordinate: pages[pdfPosition.pageNumber].styles.height - (pdfPosition.top * pages[pdfPosition.pageNumber].viewportScale),
```

Comment:
don't we need to do the same thing for the xCoordinate? does it not become relevant with rotation?

Reply:
Not from what I've seen, the y coordinate is translated as part of the rotation. the x is still a fixed distance from the margin.

Reply:
ok, if it's working when the document is rotated, then that's fine